src/vivarium_public_health/risks/base_risk.py

```diff
@@ -89,3 +89,50 @@ def get_current_exposure(self, index):
 
     def __repr__(self):
         return f"Risk(_risk_type= {self._risk_type}, _risk= {self._risk})"
+
+
+class DummyRisk(Risk):
```

Comment:
No bueno

Reply:
Yeah I don't know what's goin on here. I'm removing it